cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/GetLogic.java

```diff
@@ -36,7 +36,7 @@
      * @param servicesOperations ServicesOperations
      * @param applicationsOperations ApplicationsOperations
      * @param clientOperations ClientOperations
-     * @param loginOptions LoginCommandOptions
+     * @param loginOptions ILoginCommandOptions
```

Comment:
Should be renamed back to `LoginCommandOptions`

Reply:
Repeating the class name in the param description doesn't make a lot of sense by the way. Please consider adding an actual description of the parameter.